dirs.proj

```diff
@@ -13,7 +13,6 @@
         <!--<Solution Include="src\Microsoft.ApplicationInsights.AspNetCore\Microsoft.ApplicationInsights.AspNetCore.xproj" />-->
         <!-- Tests -->
         <TestProject Condition="$(RunTests) != '' And $(RunTests)" Include="test\Microsoft.ApplicationInsights.AspNetCore.Tests\ " />
-        <TestProject Condition="$(RunTests) != '' And $(RunTests)" Include="test\FunctionalTestUtils\ " />
```

Comment:
Because it isn't an actual test project itself but rather the utils that the other test projects use for their tests.